calico/getting-started/kubernetes/vpp/openshift.mdx

```diff
@@ -0,0 +1,211 @@
+
+```
+mkdir openshift-tigera-install && cd openshift-tigera-install
+```
+
+Run OpenShift installer to create a default configuration file:
+
+```
+openshift-install create install-config
+```
+
+:::note
+
+Refer to the [OpenShift installer documentation](https://cloud.redhat.com/openshift/install) for more information
+about the installer and any configuration changes required for your platform.
+
+:::
+
+Once the installer has finished, your staging directory will contain the configuration file `install-config.yaml`.
+
+### Update the configuration file to use {{prodname}}
+
+Override the OpenShift networking to use Calico and update the AWS instance types to meet the [system requirements](../openshift/requirements.mdx):
+
+```bash
+sed -i 's/networkType:.*/networkType: Calico/' install-config.yaml
+sed -i 's/platform: {}/platform:\n    aws:\n      type: m5.4xlarge/' install-config.yaml
+```
+
+### Generate the install manifests
+
+Generate the Kubernetes manifests using your configuration file:
+
+```bash
+openshift-install create manifests
+```
+
+<InstallOpenShiftManifests />
+
+
+Download the Calico VPP manifests for OpenShift and add them to the generated manifests directory:
+
+```bash
+mkdir vpp
+cd vpp
+curl -O "https://raw.githubusercontent.com/projectcalico/vpp-dataplane/{{vppbranch}}/yaml/platforms/openshift/00-namespace-calico-vpp-dataplane.yaml
+curl -O "https://raw.githubusercontent.com/projectcalico/vpp-dataplane/{{vppbranch}}/yaml/platforms/openshift/01-cr-installation.yaml
+curl -O "https://raw.githubusercontent.com/projectcalico/vpp-dataplane/{{vppbranch}}/yaml/platforms/openshift/03-configmap-calico-vpp-resources.yaml
+curl -O "https://raw.githubusercontent.com/projectcalico/vpp-dataplane/{{vppbranch}}/yaml/platforms/openshift/03-role-calico-vpp-dataplane.yaml
+curl -O "https://raw.githubusercontent.com/projectcalico/vpp-dataplane/{{vppbranch}}/yaml/platforms/openshift/03-rolebinding-calico-vpp-dataplane.yaml
+curl -O "https://raw.githubusercontent.com/projectcalico/vpp-dataplane/{{vppbranch}}/yaml/platforms/openshift/03-serviceaccount-calico-vpp-dataplane.yaml
+curl -O "https://raw.githubusercontent.com/projectcalico/vpp-dataplane/{{vppbranch}}/yaml/platforms/openshift/04-calico-vpp-nohuge.yaml
+cd ..
+cp vpp/* manifests/
+```
+
+Make sure that SERVICE_PREFIX in `manifests/03-configmap-calico-vpp-resources.yaml` matches the service cidr of the cluster:
+
+```bash
+SERVICE_CIDR=`grep -A1 serviceNetwork: ./manifests/cluster-config.yaml | tail -n1 | cut -d '-' -f2`
+sed -i "s#SERVICE_PREFIX:.*#SERVICE_PREFIX:$SERVICE_CIDR#" ./manifests/03-configmap-calico-vpp-resources.yaml
+```
+
+
+### Optionally provide additional configuration
+
+You may want to provide Calico or the VPP dataplane with additional configuration at install time. For example, BGP configuration or peers.
```

Comment:
s/dataplane/data plane/g

Reply:
Are you sure? The word "**dataplane**" seems to be the **norm** in most of the calico docs around the topic of different dataplanes for example 

https://docs.tigera.io/calico/latest/operations/ebpf/enabling-ebpf

Reply:
@onong "Dataplane" has been the norm, but we will soon globally change to align with industry standards.